File: demos/Game-World/assets/scripts/GameConstants.js
function GameConstant(varName, varValue) {
    print(varName + " = " + varValue);
}

//Input devices options
new GameConstant("useMouse", true);
new GameConstant("useMouseDebug", false);
new GameConstant("useMouseCursor", false);
new GameConstant("useMouseOffScreenPositionShift", true);
new GameConstant("useKeyBoard", true);
new GameConstant("useKeyBoardDebug", false);

//Input HOG2Map files options
new GameConstant("HOG2Maps", false);
new GameConstant("HOG2MapsWithScells", false);

//Geometry options
new GameConstant("showTextures", true);
new GameConstant("showWireframe", false);
new GameConstant("useDisplayLists", false);

//Geometry generators options
new GameConstant("useTerrainGenerators", true);
new GameConstant("useTerrainGeneratorsWithMiddleMinimumHeight", true);
new GameConstant("useMazeGenerators", false);
new GameConstant("useMazeGeneratorsDebug", true);

//Physics options
new GameConstant("useBroadPhaseCollisionDetection", true);
new GameConstant("useNarrowPhaseCollisionDetection", false);
new GameConstant("useLights", true);

//Scene options
new GameConstant("showGeometries", true);
new GameConstant("useUnreachableSky", false);
new GameConstant("useUnreachableFloor", false);
new GameConstant("Floor", true);
new GameConstant("SkyDome", false);
new GameConstant("SkyBox", false);

//Sound options
new GameConstant("useSoundDebug", false);

//GUI options options
new GameConstant("showFPS", true);
new GameConstant("showText", true);
new GameConstant("testMode", true);
new GameConstant("useFullScreen", false);

File: docs/demos/not_working/JGroups-v2.0/src/util/GameConstants.java
package util;

import java.util.logging.Logger;
import static java.util.logging.Logger.getLogger;

/**
 * Class to set up the game constants.
 *
 * @author G. Amador & A. Gomes
 */
public class GameConstants {

    private static final Logger log = getLogger(GameConstants.class.getName());

    /**
     * Method to set up the defaults for the game constants options.
     */
    public static void setGameConstants() {
        //Input devices options
        InputDevicesOptions.useMouse = true;
        InputDevicesOptions.useMouseCursor = false;
        InputDevicesOptions.useKeyBoard = true;

        //Geometry options
        GeometryOptions.showTextures = true;
        GeometryOptions.showWireframe = false;

        //Geometry Scene options
        GeometryOptions.Floor = false;
        GeometryOptions.SkyDome = false;
        GeometryOptions.SkyBox = false;
        GeometryOptions.addSkyBoxCeil = true;
        GeometryOptions.addSkyBoxWalls = true;

        //Geometry generators options
        GeometryOptions.useMazeGenerators = false;

        //Input HOG2Map files options
        GeometryOptions.HOG2Maps = false;

        //Physics options
        PhysicsOptions.doCollisionDetection = false;
        PhysicsOptions.doNarrowPhaseCollisionDetection = false;

        //AI options
        AIOptions.useSteeringBehaviors = true;
        AIOptions.usePathFinders = false;
        AIOptions.usePathFindersDebug = false;
        AIOptions.usePathFindersTimeout = false;
        AIOptions.usePathFindersGraphHashMap = false;
        AIOptions.usePathFindersPotentialFields = false;

        //Extras options
        ExtrasOptions.useLights = true;

        //TODO: read from file        
    }

    /**
     * Default constructor.
     */
    GameConstants() {

    }
}
